src/components/Header/Header.styles.css

```diff
@@ -0,0 +1,49 @@
+.header-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+/***************MENU TOGGLE****************/
+.header-icon {
+  height: 50px !important;
```

Comment:
Avoid using `!important`. It is a bad practice. If there are no more elements to add specificity to the selector, you can duplicate the class name like so:
```
.header-icon.header-icon {
  height: 50px;
...
```